lib/pf/Portal/Profile.pm

```diff
@@ -511,6 +511,28 @@ sub findScan {
     return undef;
 }
 
+=item getManadoryFieldsForSources
```

Comment:
I'd also renamed that for 'getFieldsForSources' since it returns both the custom configured fields and the source mandatory fields...